frontend/src/libs/components/modal/steps/delete-account/libs/types/delete-account-form-payload.ts

```diff
@@ -0,0 +1,11 @@
+type DeleteAccountFormPayload = {
```

Comment:
I think it should be stored in shared because mobile side also has to log out some way)